src/Operation/UpdateSearchIndex.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/*
+ * Copyright 2015-present MongoDB, Inc.
```

Comment:
New copyright blocks should use the current year. I'll defer to you to update other occurrences.

```suggestion
 * Copyright 2023-present MongoDB, Inc.
```